src/Psalm/Internal/Fork/PsalmRestarter.php

```diff
@@ -74,16 +74,25 @@ protected function restart(array $command): void
 
             file_put_contents($this->tmpIni, $content);
         }
+
+        $additional_options = [];
+
+        // executed in the parent process (before restart)
+        // if it wasn't loaded then we apparently don't have opcache installed and there's no point trying
+        // to tweak it
+        if (extension_loaded('opcache') || extension_loaded('Zend OPcache')) {
```

Comment:
`Zend OPcache` is what the extension called on my system:

```console
$ php -r 'var_dump(["opcache" => extension_loaded("opcache"), "Zend OPcache" => extension_loaded("Zend OPcache")]);'
array(2) {
  'opcache' =>
  bool(false)
  'Zend OPcache' =>
  bool(true)
}
```

Reply:
I think using the `opcache_get_status` function is more reliable, as I did in https://github.com/vimeo/psalm/pull/9240/files

Reply:
Actually it may be less reliable as it can throw warnings depending on the configuration: https://www.php.net/opcache_get_status#refsect1-function.opcache-get-status-errors